backend/models/service_model.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel, Field
+from enum import Enum
+from typing import List
+from datetime import datetime
+from typing import Optional
+from .enum_for_models import ProgramTypeEnum
+
+
+class Service(BaseModel):
+    id: int | None = None
+    created_at: datetime | None = None
```

Comment:
For consistency: either created_at: Optional[datetime] or created_at: datetime | None = None.